src/components/pet-details/PetMedical.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TablePagination,
+  Paper,
+  Typography,
+} from "@mui/material";
+
+const PetMedical = (props: { medical: any[] }) => {
+  const { medical } = props;
+  const [page, setPage] = React.useState(0);
+  const [rowsPerPage, setRowsPerPage] = React.useState(5);
+
+  const handleChangePage = (event: unknown, newPage: number) => {
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ) => {
+    setRowsPerPage(+event.target.value);
+    setPage(0);
+  };
+
+  const dateFormat = (date: string) => {
+    const match = date.match(
```

Comment:
עדיף בכללי להשתמש בספרייה אחידה לאפליקציה לכל התאריכים ולדוגמא דרך מומטנט שממש טובה https://momentjscom.readthedocs.io/en/latest/moment/01-parsing/15-is-valid/

Reply:
שמתי TODO